indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -239,6 +249,7 @@
             :short-name.lowercase (util/safe-lowercase short-name)
             :version-id version-id
             :version-id.lowercase (util/safe-lowercase version-id)
+            :parsed-version-id.lowercase (util/safe-lowercase parsed-version-id)
```

Comment:
Not sure of the history on this and why it was originally done this way. I haven't seen anything in the elasticsearch documentation that would need it this way, but I am wondering if it's something elasticsearch specific that I am missing.